README.md

```diff
@@ -8,6 +8,7 @@ To quickly learn how to run cleanlab on your own data, first check out the [quic
 
 |     | Example                                                                                        | Description                                                                                                                                                                                                                                                                  |
 | --- | ---------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0   | [datalab](datalab_image_classification)                              | Use Datalab to detect issues in (a subset of) the Caltech-256 image classification dataset.                        |
```

Comment:
TODO: @elisno 

Please rename the entire folder where all these files live to be: `datalab_image_classification` as I've specified in this table of contents link here.